packages/desktop-client/e2e/transactions.test.ts

```diff
@@ -163,6 +163,11 @@ test.describe('Transactions', () => {
     await expect(transaction.category).toHaveText('Transfer');
     await expect(transaction.debit).toHaveText('12.34');
     await expect(transaction.credit).toHaveText('');
+
+    // Wait for balance calculations to complete before taking screenshot
+    await page.waitForLoadState('networkidle');
```

Comment:
Instead of a time-based wait, is it possible to get the balance values before entering transaction and asserting after adding?

e.g. 
```javascript
const balance = accountPage.getBalance();

await accountPage.addEnteredTransaction();
...

const newBalance = accountPage.getBalance();

// assert balance !== newBalance

```